src/api/app/controllers/webui/projects/bs_requests_controller.rb

```diff
@@ -4,12 +4,16 @@ class BsRequestsController < WebuiController
       before_action :set_project
 
       def index
-        parsed_params = BsRequest::DataTable::ParamsParserWithStateAndType.new(params).parsed_params
-        requests_query = BsRequest::DataTable::FindForProjectQuery.new(@project, parsed_params)
-        @requests_data_table = BsRequest::DataTable::Table.new(requests_query, params[:draw])
+        if Flipper.enabled?(:request_index, User.session)
+          redirect_to project_requests_beta_path(@project, involvement: 'incoming', state: %w[new review])
```

Comment:
Hm... why do you do this decision here and not in the place you link?

Reply:
In the UI we replaced the links to point to this endpoint (we just pushed again, we accidentally pointed there also to the beta path). The want to refactor this in small chunks, so the idea is to first point everywhere in the UI to the new route, keep the redirect here. Afterwards move the controller code and get rid of the redirect here. This way do not have to change the URLs twice.

Reply:
okay...